aten/src/ATen/native/mps/operations/Inverse.mm

```diff
@@ -0,0 +1,75 @@
+#include <ATen/ATen.h>
+#include <ATen/Tensor.h>
+#include <ATen/Utils.h>
+#include <ATen/mps/MPSStream.h>
+#include <ATen/native/mps/OperationUtils.h>
+#include <torch/library.h>
+#include <c10/util/Optional.h>
+
+
+namespace at {
+namespace native {
+
+TORCH_IMPL_FUNC(linalg_inv_ex_out_mps)(const Tensor& A, bool check_errors, const Tensor& result, const Tensor& info)
+{
+    TORCH_CHECK(result.is_mps(), "Output tensor is not MPS");
+    Tensor output = result.clone().contiguous();
```

Comment:
Do we need to make the result contiguous here? can you check what other backends are doing.